clean_project/test_data/scriptdb.sql

```diff
@@ -0,0 +1,13 @@
+-- Create a database
+CREATE DATABASE index_db
+-- Create user with a specific option
+CREATE USER gianluca
+-- Grant all privileges on a specific database to given user
+grant all privileges on database index_db to gianluca;
+-- Grant SELECT, INSERT, UPDATE, DELETE on all tables to given user
+GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA public to gianluca;
+-- Grant all default privileges on a specific schema about tables and sequences to given user
+ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT ALL PRIVILEGES ON TABLES TO gianluca;
+ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT ALL PRIVILEGES ON sequences TO gianluca;
+-- Create a table with a specific fields
+CREATE TABLE index(Word INTEGER, File text, Occurrence integer[]);
```

Comment:
This table is fine, but like this it's going to be very slow when heavy with data.
It would be great if you put the words in a (database) index so that you could do fast lookup. You can do that manually (create index word_index etc.) or you can declare the word as a primary key, that will make an index for them automatically. 

Both of these things will require the word to be unique in the table. If that's not the case there are ways to work around that, but this is not the case, innit?